api/src/pcapi/core/finance/models.py

```diff
@@ -719,3 +725,98 @@ class PaymentMessage(Base, Model):
     id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
     name: str = sqla.Column(sqla.String(50), unique=True, nullable=False)
     checksum: bytes = sqla.Column(sqla.LargeBinary(32), unique=True, nullable=False)
+
+
+##
+# When an incident is `CREATED` it can be either `CANCELLED` and it's the end of this incident. Or it can be `VALIDATED`.
+# `TERMINATED` means the incident is finished and all the payments completed
+##
+class IncidentStatus(enum.Enum):
+    CREATED = "created"
+    VALIDATED = "validated"
+    TERMINATED = "terminated"
```

Comment:
Il me semble qu'on n'avait décidé de ne conserver que les états de validation (par l'équipe comptabilité). Donc `TERMINATED` n'a plus lieu d'être, non ?

Reply:
Pour permettre la vérification qu'une réservation n'est pas déjà impliquée dans un incident déjà créé, il faut pouvoir avoir le statut `Terminated` afin de délimiter la possibilité qu'une réservation soit impliquée dans un incident.

Reply:
Je ne comprends pas : 
- Que représente cet état ? Qu'est-ce qui déclenche le passage dans cet état ?
- Pourquoi cet état est nécessaire ? Pour savoir si une réservation est liée à un incident, on peut regarder si la requête suivante renvoie quelque chose : `SELECT finance_incident.id from finance_incident join finance_incident_booking on ... where finance_incident_booking."bookingId" = :booking_id and finance_incident.status != 'CANCELLED'` , non ?